notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.        .sum()
</pre><p>I think we can add <code>.astype(float)</code> after this so we don't need to say <code>df.astype(float)</code> everywhere else below  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>